07LessonSeven/in-class-project/css/my-style.css

```diff
@@ -12,7 +12,7 @@ h1 {
   color: white;
```

Comment:
Try and center your NASA page with some style on the body - margin: 0 auto; and width: 90%;.